src/main/groovy/org/broadinstitute/orsp/CollectionLinkStatus.groovy

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.orsp
+
+enum CollectionLinkStatus {
+
+
+    Approved("Approved"),
```

Comment:
Enum instances are constants and should follow the all uppercase naming convention ... (see https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html)
```suggestion
    APPROVED("Approved"),
```